parser/Builder.cc

```diff
@@ -230,11 +262,28 @@ class Builder::Impl {
                 auto decls = driver_->alloc.node_list();
                 decls->emplace_back(toForeign(std::move(intro)));
                 driver_->numparam_stack.regis(name_str[1] - '0', std::move(decls));
-            }
+            } else if (isItParameterName(name_str) && driver_->lex.context.allowNumparams &&
```

Comment:
Might be worth naming the result of `isItParameterName` up above line 209, and reusing that result down here.

Reply:
Good call, done